### ISSUE TITLE
the code of WQMIX misses the feed-forward network for Q*  estimation

### ISSUE THREAD

**Issue by ftpot** [oxwhirl/wqmix#7]
![compare code with paper ](https://user-images.githubusercontent.com/87177796/186121889-95a05f66-1331-4aca-92bd-c757d44367f9.png)

**Comment by Hemsworth** [comment]
Yes, this is not the same as the implementation in the paper, can the author answer it?

**Comment by lllllin27** [comment]
i have same question.the Q* network also have abs function in author code ,and update periodically from eval network in author's code.
